### ISSUE TITLE
The training process was stuck when training using my own dataset

### ISSUE THREAD

**Issue by Kongsea** [sfzhang15/RefineDet#130]
When I train the refinedet network using my own dataset, the terminal outputs the following information and stuck here:

> ......
> I1108 16:06:45.567307 25088 net.cpp:226] res2b_branch2a needs backward computation.
> I1108 16:06:45.567311 25088 net.cpp:226] res2a_res2a_relu_0_split needs backward computation.
> I1108 16:06:45.567312 25088 net.cpp:226] res2a_relu needs backward computation.
> I1108 16:06:45.567315 25088 net.cpp:226] res2a needs backward computation.
> I1108 16:06:45.567318 25088 net.cpp:226] scale2a_branch2c needs backward computation.
> I1108 16:06:45.567320 25088 net.cpp:226] bn2a_branch2c needs backward computation.
> I1108 16:06:45.567323 25088 net.cpp:226] res2a_branch2c needs backward computation.
> I1108 16:06:45.567325 25088 net.cpp:226] res2a_branch2b_relu needs backward computation.
> I1108 16:06:45.567328 25088 net.cpp:226] scale2a_branch2b needs backward computation.
> I1108 16:06:45.567330 25088 net.cpp:226] bn2a_branch2b needs backward computation.
> I1108 16:06:45.567333 25088 net.cpp:226] res2a_branch2b needs backward computation.
> I1108 16:06:45.567335 25088 net.cpp:226] res2a_branch2a_relu needs backward computation.
> I1108 16:06:45.567338 25088 net.cpp:226] scale2a_branch2a needs backward computation.
> I1108 16:06:45.567340 25088 net.cpp:226] bn2a_branch2a needs backward computation.
> I1108 16:06:45.567342 25088 net.cpp:226] res2a_branch2a needs backward computation.
> I1108 16:06:45.567345 25088 net.cpp:226] scale2a_branch1 needs backward computation.
> I1108 16:06:45.567348 25088 net.cpp:226] bn2a_branch1 needs backward computation.
> I1108 16:06:45.567350 25088 net.cpp:226] res2a_branch1 needs backward computation.
> I1108 16:06:45.567353 25088 net.cpp:226] pool1_pool1_0_split needs backward computation.
> I1108 16:06:45.567355 25088 net.cpp:226] pool1 needs backward computation.
> I1108 16:06:45.567358 25088 net.cpp:226] conv1_relu needs backward computation.
> I1108 16:06:45.567360 25088 net.cpp:226] scale_conv1 needs backward computation.
> I1108 16:06:45.567363 25088 net.cpp:226] bn_conv1 needs backward computation.
> I1108 16:06:45.567365 25088 net.cpp:226] conv1 needs backward computation.
> I1108 16:06:45.567368 25088 net.cpp:228] label_data_1_split does not need backward computation.
> I1108 16:06:45.567373 25088 net.cpp:228] data_data_0_split does not need backward computation.
> I1108 16:06:45.567376 25088 net.cpp:228] data does not need backward computation.
> I1108 16:06:45.567382 25088 net.cpp:270] This network produces output arm_loss
> I1108 16:06:45.567386 25088 net.cpp:270] This network produces output odm_loss
> I1108 16:06:45.567644 25088 net.cpp:283] Network initialization done.
> I1108 16:06:45.568662 25088 solver.cpp:75] Solver scaffolding done.
> I1108 16:06:45.600242 25088 caffe.cpp:155] Finetuning from models/ResNet/ResNet-101-model.caffemodel
> I1108 16:06:45.708572 25088 upgrade_proto.cpp:67] Attempting to upgrade input file specified using deprecated input fields: models/ResNet/ResNet-101-model.caffemodel
> I1108 16:06:45.708595 25088 upgrade_proto.cpp:70] Successfully upgraded file specified using deprecated input fields.
> W1108 16:06:45.708598 25088 upgrade_proto.cpp:72] Note that future Caffe releases will only support input layers and not input fields.
> I1108 16:06:45.736232 25088 net.cpp:761] Ignoring source layer pool5
> I1108 16:06:45.736253 25088 net.cpp:761] Ignoring source layer fc1000
> I1108 16:06:45.736255 25088 net.cpp:761] Ignoring source layer prob
> I1108 16:06:45.786772 25088 parallel.cpp:392] GPUs pairs 0:1
> I1108 16:06:46.092105 25088 annotated_data_layer.cpp:62] output data size: 1,3,512,512
> I1108 16:06:48.305120 25088 parallel.cpp:425] Starting Optimization
> I1108 16:06:48.305336 25088 solver.cpp:294] Solving mars_refinedet_resnet101_512x512_train
> I1108 16:06:48.305343 25088 solver.cpp:295] Learning Rate Policy: multistep
> I1108 16:06:48.377758 25088 blocking_queue.cpp:50] Data layer prefetch queue empty

So I want to know what happens and how to fix it and let the training process continue, thank you.

**Comment by sfzhang15** [comment]
@Kongsea 
Sorry about the late respond.
If you use VOC or COCO to train, still have this problem? From these message, I do not know  what happens.
